File: 0242-valid-anagram/0242-valid-anagram.ex
defmodule Solution do
      @spec is_anagram(s :: String.t, t :: String.t) :: boolean
      def is_anagram(s, t) do
        String.equivalent?(sort(s), sort(t))
      end
      
    defp sort(str) do
        String.to_charlist(str)
        |> Enum.sort()
        |> List.to_string()
    end
end

File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length()) return false;
        unordered_map<char, int> map;
        for (int i = 0; i < s.length(); i++)
        {
            map[s[i]]++;
            map[t[i]]--;
        }
        for (auto count : map) if (count.second) return false;
        return true;
    }
};